ex5.sh

```diff
@@ -0,0 +1,13 @@
+touch subs/File{A..F}.txt

```

Comment:
use && to ensure that the second command executes only if the first command succeeds.